content/articles/how-to-send-an-email-with-django/index.md

```diff
@@ -0,0 +1,259 @@
+description: This tutorial is focused on teaching you how to send an email with Django using the Google SMTP service provider.
+author: samuel-torimiro
+date:
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/how-to-send-an-email-with-django/hero.jpg
+    alt: Gmail example image
+---
+
+In most web applications that are customer-driven, there may be a need to send an email. Sending an email with Django is quite simple. In the first part of this tutorial, we will quickly set up a new Django project from scratch with some URLs, views and templates. Then, we will take a look at configuring the Google SMTP service provider and finally how to use the `send_mail()` function Django provides, to enable us to send emails.
+
+### Prerequisites
+To follow along with this tutorial, you will need a Google account. If you don't have one follow this [link](https://accounts.google.com/signup/v2/webcreateaccount?flowName=GlifWebSignIn&flowEntry=SignUp) to create a new Google account.
+
+Prior knowledge of Django is also required.
+
+### Project Setup and Overview
+Open your terminal or command prompt, and run the following commands:
+
+```bash
+mkdir mail && cd mail
+
+pipenv install django
+
+pipenv shell
+```
+
+We created a new virtual environment using the `pipenv` command, you can install Pipenv if you don't have it already installed using `pip install pipenv`.
+
+Next, create a new Python project and application.
+
+```bash
+django-admin startproject send_mail .
+
+python manage.py migrate
+
+python manage.py startapp mailer
+
+python manage.py runserver
+```
+
+Next, navigate to <http://127.0.0.1:8000/> with your browser, you should see the default Django homepage.
+
+![Django homepage](/engineering-education/how-to-send email-with-django/django-homepage.jpg)
+
+Next, add the new application to your `settings.py` file.
+
+```python
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+
+    # new
+    'mailer.apps.MailerConfig',
+]
+```
+
+Next, update your project `send_mail/urls.py` to look like this:
+
+```python
+from django.contrib import admin
+from django.urls import path, include # new
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('', include('mailer.urls')), # new
+]
+```
+
+Next, create a new file inside the `mailer` directory called `urls.py` and add the following code:
+
+```python
+from django.urls import path
+from .views import sendMail
+
+urlpatterns = [
+    path('', sendMail),
+]
+```
+
+Next, create a new file in your `mailer` directory called `forms.py` and add the following code:
+
+```python
+from django import forms
+
+class EmailForm(forms.Form):
+    name = forms.CharField(max_length=50)
+    email = forms.EmailField()
+    to = forms.EmailField()
+    message = forms.CharField(widget=forms.Textarea)
```

Comment:
@Samuel-2626 Please remove the unused form entries. `email` and `name` are not used in views.py. Also, `to` is a very short variable name. Consider changing it to be more descriptive.